packages/client/src/client/FinchClient.ts

```diff
@@ -134,10 +141,27 @@ export class FinchClient {
         options.messageKey ?? this.messageKey,
         !!this.id,
       );
-
+      const cancel = () => {
```

Comment:
i'm trying to see where this is being called in order to set `cancelled` to `true`. i do see where it's being added and deleted but not executed.

Reply:
It's being canceled [here](https://github.com/jointoucan/finch-graphql/pull/307/files#diff-5a2be6b8f003128861aaa3d41009fa0806ad7bf20546bb5aa5ea801b32ce42faR185). Essentially wanted to send some type of response to queries that are potentially still pending when this happens.